roofit/xroofit/src/xRooFit.cxx

```diff
@@ -921,28 +1219,43 @@ xRooFit::minimize(RooAbsReal &nll, const std::shared_ptr<ROOT::Fit::FitConfig> &
 
       // call minos if requested on any parameters
       if (status == 0 && minos) {
-         std::unique_ptr<RooAbsCollection> pars(floatPars->selectByAttrib("minos", true));
-         for (auto p : *pars) {
-            xRooFit::minos(nll, *out, p->GetName(), myFitConfig);
+         for(auto label : {"xminos","xMinos"}) {
+            std::unique_ptr<RooAbsCollection> pars(floatPars->selectByAttrib(label, true));
+            for (auto p: *pars) {
+               Info("minimize", "Computing xminos error for %s", p->GetName());
+               xRooFit::minos(nll, *out, p->GetName(), myFitConfig);
+            }
+            if (!pars->empty())
+               *floatPars = out->floatParsFinal(); // put values back to best fit
          }
-         if (!pars->empty())
-            *floatPars = out->floatParsFinal(); // put values back to best fit
       }
 
-      if (restore) {
-         *floatPars = out->floatParsInit();
-      }
    }
+
+   if (restore) {
+      *floatPars = out->floatParsInit();
+   }
+
    if (out && !logs.empty()) {
       // save logs to StringVar in constPars list
-      const_cast<RooArgList &>(out->constPars()).addOwned(std::make_unique<RooStringVar>(".log", "log", logs.c_str()));
+      const_cast<RooArgList &>(out->constPars()).addOwned(*new RooStringVar(".log", "log", logs.c_str()));
```

Comment:
```suggestion
#if ROOT_VERSION_CODE >= ROOT_VERSION(6, 28, 00)
      const_cast<RooArgList &>(out->constPars()).addOwned(std::make_unique<RooStringVar>(".log", "log", logs.c_str()));
#else
const_cast<RooArgList &>(out->constPars()).addOwned(*new RooStringVar(".log", "log", logs.c_str()));
#endif
```
Also, the `addOwned()` overload that gets a raw pointer is considered a memory unsafe interface that can't be used in the ROOT repo anymore. However, the memory safe alternative is only available since ROOT 6.28:
https://github.com/root-project/root/commit/2f6c936dd247687d60a88c7acd2ab39180e16eb6

So for backwards compatibility of the standalone xroofit, you might want to add this preprocessor branch.